### PR TITLE
feat(x/launch/keeper): verify and validate all request contents

### LINKED ISSUE #232: Verify and validate all request content
Verify and validate all request content interfaces
https://github.com/tendermint/spn/blob/develop/x/launch/types/request_content.go

e.g.:
```go
// RequestContent defines the interface for a request content
type RequestContent interface{
	Validate() error
}

var _ RequestContent = &AccountRemoval{}

// Validate check the AccountRemoval object
func (g AccountRemoval) Validate() error {
	...
	return nil
}
```

### PR DESCRIPTION
**Pull request by Pantani**
Closes #232

### What does this PR does?

Create a request content interface validation.

### How to test?

```shell
$ make test
```

### CODE REVIEW

File: x/launch/types/request_content_test.go [hunk 1]
```diff
@@ -125,5 +125,333 @@ func TestVestedAccountCodec(t *testing.T) {
...
+			name: "invalid chain id",
+			content: types.VestedAccount{
+				Address:         sample.AccAddress(),
+				ChainID:         "invalid_chain",
+				StartingBalance: sample.Coins(),
+				VestingOptions:  option,
+			},
+			wantErr: true,
+		}, {
+			name: "nil message option",
```
Comment (lubtd) [inline x/launch/types/request_content_test.go:411]:
```suggestion
			name: "no vesting option",
```

File: x/launch/types/request_content_test.go [hunk 2]
```diff
@@ -125,5 +125,333 @@ func TestVestedAccountCodec(t *testing.T) {
...
+			name: "invalid coins",
+			content: types.VestedAccount{
+				Address:         sample.AccAddress(),
+				ChainID:         chainID,
+				StartingBalance: sdk.Coins{sdk.Coin{Denom: "", Amount: sdk.NewInt(10)}},
+				VestingOptions:  option,
+			},
+			wantErr: true,
+		}, {
+			name: "invalid message option",
```
Comment (lubtd) [inline x/launch/types/request_content_test.go:429]:
```suggestion
			name: "invalid vesting option",
```

File: x/launch/types/request_content_test.go [hunk 3]
```diff
@@ -125,5 +125,333 @@ func TestVestedAccountCodec(t *testing.T) {
...
+			if tt.wantErr {
+				require.Error(t, err)
+				return
+			}
+			require.NoError(t, err)
+		})
+	}
+}
+
+func TestGenesisAccount_Validate(t *testing.T) {
```
Comment (lubtd) [inline x/launch/types/request_content_test.go:161]: I will not list all of them but several time we talk about "message" in error description like "valid message"

We should rename message to request

**Review by lubtd — COMMENTED** [review]
Looks legit to me! Just some part I think we should do some renaming.

We will need to merge this one before finishing https://github.com/tendermint/spn/pull/250 oneof with requests, otherwise that's gonne be a conflict nightmare

**Review by lubtd — APPROVED** [review]

**Review by ilgooz — APPROVED** [review]

**Pull request merged**